CalibTracker/SiStripCommon/test/testProduceCalibrationTree_cfg.py

```diff
@@ -94,13 +95,22 @@
 ## process.L1T1.L1TechTriggerSeeding = cms.bool(True)
 ## process.L1T1.L1SeedsLogicalExpression = cms.string('(40 OR 41) AND NOT (36 OR 37 OR 38 OR 39)')
 
-process.EventInfo = cms.EDAnalyzer("ShallowTree",
-                                   outputCommands = cms.untracked.vstring('drop *',
+compressionSettings = 201
```

Comment:
@echabert out of curiosity is there any documentation about the compression settings ? It would be nice to put a comment in the code to explain what one expects from this change, maybe a pointer to the root user manual. 
I am just playing devil's advocate here, the release team is in general against "magic number" in the code. Thanks!

Reply:
@mmusich Good point. Here are some links and additional info: 
  - https://root.cern.ch/root/html534/TFile.html#TFile:SetCompressionSettings
   -->  settings = 100 * algorithm + level 
    -  level is from 1 to 9 
    -  algo: 1 (ZLIB), 2 (LMZA)
  - compression & performance: https://root.cern.ch/root/html534/guides/users-guide/InputOutput.html#compression-and-performance